Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert_classifier/deploy.py

```diff
@@ -0,0 +1,18 @@
+import mlflow
+from mlflow.deployments import get_deploy_client
+
+
+def deploy():
+    mlflow.set_tracking_uri("http://localhost:5005")
```

Comment:
I think that it would be better using MLFlow CLI to deploy. Going for MLFlow Python API maybe would result in a better self-contained solution in the future (if we create a plugin or extension) but it would need some time.

Reply:
Will change it

Reply:
Changed to use the CLI in the makefile. Left this to try out the Python API if anyone wants